src/docs/010_manual/60_further_reading.adoc

```diff
@@ -6,102 +6,100 @@ ifndef::imagesdir[:imagesdir: ../images]
-AsciiDoc is our preferred markup language for technical docs
+.AsciiDoc
+[%collapsible]
+==== 
+This is our preferred markup language for technical docs.
+
 ** https://asciidoctor.org/docs/user-manual/[Asciidoctor User-Manual]
 ** https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/[AsciiDoc Syntax Quick Reference]
-** Video: https://www.ahus1.de/post/asciidoctor-intro-and-deep-dive["Asciidoctor Deep Dive"] by Alexander Schwartz
-* *Markdown* +
-Since we use jBake as Static-Site-Generator, you can also use Markdown to write your docs.
-** https://github.com/vsch/flexmark-java[flexmark-java] is the Markdown parser and flavour used by jBake
-** if you prefer to use the xref:../015_tasks/03_task_exportMarkdown.adoc[`exportMarkdown`]-Task, then https://github.com/bodiam/markdown-to-asciidoc[markdown-to-asciidoc] library is used. The IntelliJ-Plugin also uses this library.
+** https://www.ahus1.de/post/asciidoctor-intro-and-deep-dive["Asciidoctor Deep Dive Video"] by Alexander Schwartz
+
+====
+
+
+.Markdown
+[%collapsible]
+==== 
+Since we use jBake as Static-Site-Generator, you can write your docs in Markdown too.
+
+** https://github.com/vsch/flexmark-java[flexmark-java] is the Markdown parser and flavor jBake uses
+** https://github.com/bodiam/markdown-to-asciidoc[markdown-to-asciidoc] library can be used if you prefer to use xref:../015_tasks/03_task_exportMarkdown.adoc[`exportMarkdown`]-Task. 
+
+====
 
 === Templates
+.arc42
+[%collapsible]
+==== 
+
+The perfect template for your software solution architecture documentation.
 
-* *arc42* +
-is the perfect documentation template for your softare solution architecture documentation.
 ** https://arc42.org/[arc42]
 ** https://docs.arc42.org/home/[arc42 Tips & Tricks]
 ** https://faq.arc42.org/home/[arc42 FAQ]
+==== 
 
 === Docs-as-Code
+.docs-as-code
+[%collapsible]
+==== 
+
+Treat your docs as if they were code.
 
-* *docs-as-code* +
-is the approach you use when you treat your docs the same as your code.
-We've create a little microsite with some additional resources.
 ** https://docs-as-co.de/[Docs-as-Code]
-** https://www.writethedocs.org/guide/docs-as-code/[write the docs: docs-as-code] is another independent resource.
+** https://www.writethedocs.org/guide/docs-as-code/[write the docs: docs-as-code]
+==== 
 
 === Static Site Generator
+.Static site Generator
+[%collapsible]
+==== 
 
-* https://jbake.org[jBake] is the underlying static site generator for the xref:../015_tasks/03_task_generateSite.adoc[`generateSite`]-Task
-* https://www.docsy.dev/[docsy] is the standard theme we use
-* https://getbootstrap.com/docs/4.6/getting-started/introduction/[twitter bootstrap] is the CSS framework
+* The underlying static site generator for the xref:../015_tasks/03_task_generateSite.adoc[`generateSite`]-Task: https://jbake.org[jBake]
+* Our standard theme: https://www.docsy.dev/[docsy] 
+* Our CSS framework: https://getbootstrap.com/docs/4.6/getting-started/introduction/[twitter bootstrap] 
```

Comment:
Change to:

Our CSS framework is https://getbootstrap.com/docs/4.6/getting-started/introduction/[twitter bootstrap].